openapi-contracts/src/main/com/sinch/sdk/domains/voice/models/dto/v1/DestinationTypeDto.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Voice API | Sinch
+ * The Voice API exposes calling- and conference-related functionality in the Sinch Voice Platform.
+ *
+ * The version of the OpenAPI document: 1.0.1
+ * Contact: support@sinch.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sinch.sdk.domains.voice.models.dto.v1;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Can be of type &#x60;number&#x60; for PSTN endpoints or of type &#x60;username&#x60; for data
+ * endpoints.
+ */
+public enum DestinationTypeDto {
+  NUMBER("number"),
+
+  NUMBER2("Number"),
```

Comment:
😭 I'm sure we can find a better name, or maybe convert the value in lower case before trying to match

Reply:
I agree but this files are coming from the open api generator tools so, currently,  this is how it generate names and at mustache template level we do not a lot of possibility to change it.
On the other side, we cannot lower/upper too much at runtime: if server is accepting lowercase as input and return uppercase in some other situation. This is having to be handled from context execution (this is the adapter layers responsibility)